cli/command/formatter/container.go

```diff
@@ -115,7 +116,22 @@ func (c *containerContext) Image() string {
 		if trunc := stringid.TruncateID(c.c.ImageID); trunc == stringid.TruncateID(c.c.Image) {
 			return trunc
 		}
+		// truncate digest if no-trunc option was not selected
+		ref, err := reference.ParseNormalizedNamed(c.c.Image)
+		if err == nil {
+			if nt, ok := ref.(reference.NamedTagged); ok {
+				// case for when a tag is provided
+				if namedTagged, err := reference.WithTag(reference.TrimNamed(nt), nt.Tag()); err == nil {
+					return reference.FamiliarString(namedTagged)
+				}
+			} else {
+				// case for when a tag is not provided
+				named := reference.TrimNamed(ref)
+				return reference.FamiliarString(named)
```

Comment:
@aaronlehmann I see your point. Although, given that `--no-trunc` exists, it's not the case that there's no way to easily retrieve full information. Wouldn't the same reasoning apply to `service ps/ls` too? Anyhow, I'm happy to discuss the UX implications of this. Not sure who else to tag here.

(My personal opinion is that I find it cleaner to not have to look at digests unless I specifically ask for them. @cpuguy83 indicated the same earlier. This also came up in another issue that I can't find right now.)